salt/pillar/journal-cms.sls

```diff
@@ -9,6 +9,12 @@ journal_cms:
         user: elife_1_0
         password: legacy_cms
 
+    logs:
+        file_path: private://monolog/
```

Comment:
from a first look provisioning the .yml file seems correct, but I fear what happens with the folders like these one (don't know when/where they are created). We'll test this together using a builder Vagrant VM.